_data/whats-new.yml

```diff
@@ -4,8 +4,64 @@ description: |
   We exclude from this list proofreading, spelling checks, and all minor updates.
 link: /whats-new.html
 thread: /whatsnew-feed.xml
-updated: Fri Aug 9 23:29:50 2019
+updated: Mon Aug 19 16:06:34 2019
 entries:
+- description: Added a tutorial about how to create a [grouped product using the REST
+    API](https://devdocs.magento.com/guides/v2.3/rest/tutorials/grouped-product/create-and-manage-grouped-products.html).
+  versions: 2.3.x
+  type: New topic
+  date: August 19, 2019
+  link: https://github.com/magento/devdocs/pull/5155`1  
```

Comment:
redundant symbols in the pull request number: `5155`1  `

Reply:
Do not edit the data file manually to avoid such issues.
Regenerate the file after all the pull requests with issues were updated by responsible writers.